Bookstore - Add/remove/edit function

ReshyResh opened the pull request:
# Project requirements
  - Implement function to add/remove book, updating the state.
  - Add function to edit the percentage of completion of the book ( Currently using prompt, will be updated to modal)

# Screenshot

![](https://i.imgur.com/02AYuBS.png)

wandji20 reviewed (changes requested):
Hello, @ReshyResh 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark: Nice implementation of the edit functionality

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .eslintrc.json, etc).
- [ ] It will be nice to customize your README file.

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ReshyResh commented:
As requested, I added the local React state for the input when adding a new book and changed the `deleteBook` function to only take the ID of the book, not the whole object.

State demonstration: 

![](https://i.imgur.com/beqIlr0.gif)

Emmyn5600 commented on src/components/AddBook.js:
```diff
@@ -12,24 +12,24 @@ const AddBook = () => {
-        <input type="text" name="title" placeholder="title" />
-        <input type="text" name="author" placeholder="author" />
+        <input type="text" name="title" placeholder="Title" />
+        <input type="text" name="author" placeholder="Author" />
```
Well done

Emmyn5600 commented on src/components/BookInfo.js:
```diff
@@ -14,7 +14,6 @@ const BookInfo = (props) => {
       <p className="book-title">{title}</p>
       <p className="book-author">{author}</p>
-      <button className="delete-book" type="button"> Comments </button>
       <button
```
Well done

Emmyn5600 reviewed (approved):
Hi @ReshyResh  

## STATUS: APPROVED :muscle::clap:

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

 #### Optional suggestions
 
   - N/A
   
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
